packages/example/src/pages/components/InlineNotification.mdx

```diff
@@ -6,7 +6,7 @@ description: Usage instructions for the Inline notification component
 <PageDescription>
 
 The `<InlineNotification>` component is used to communicate important
-information to a user.
+information to a user. For accessibility reasons, `<InlineNotification>` content cannot include interactive elements, just text.
```

Comment:
`ActionableNotification` accepts an `inline` prop that can be used here. It should mostly be a drop-in replacement from a usage perspective.